index.d.ts

```diff
@@ -625,11 +675,15 @@ export interface Visitor {
   'Conditional:exit'?: (node: Conditional) => any;
   'IndexAccess:exit'?: (node: IndexAccess) => any;
   'MemberAccess:exit'?: (node: MemberAccess) => any;
+  'HexNumber:exit'?: (node: HexNumber) => any;
+  'DecimalNumber:exit'?: (node: DecimalNumber) => any;
+  'Break:exit'?: (node: Break) => any;
+  'Continue:exit'?: (node: Continue) => any;
 }
 export interface ParserOpts {
   tolerant?: boolean;
   range?: boolean;
   loc?: boolean;
 }
 export function parse(sourceCode: string, parserOpts: ParserOpts): ASTNode;
-export function visit(ast: ASTNode, visitor: Visitor): void;
+export function visit(ast: ASTNode | ASTNode[] | null | undefined, visitor: Visitor): void;
```

Comment:
Visit only officially accepts non-null, non-array AST nodes here. The behavior for other cases remains undefined.

Reply:
Should list of ASTNode be accepted?

It will simplify code like
```
visitBlock(node: Block) {
  visit(node.statements, visitor)
}
```

Reply:
Why not just visit the whole Block node in that case? Writing the above kind of code couples the users much more to the internal structure of the AST.

Reply:
I see. Thanks.